src/pages/patientView/PatientViewPage.tsx

```diff
@@ -36,6 +37,13 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
             );
         }
 
+        let mutationDiv: Element | null = document.getElementById('mutations_div_prototype');
```

Comment:
this isn't necessary.  i can walk you through how to get the component on the page/expose it to legacy project.